src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-page.component.ts

```diff
@@ -109,6 +109,8 @@ export class InstructorSessionResultPageComponent extends InstructorCommentsComp
   FeedbackSessionPublishStatus: typeof FeedbackSessionPublishStatus = FeedbackSessionPublishStatus;
   isExpandAll: boolean = false;
 
+  readonly TIME_FORMAT: string = 'ddd, DD MMM, YYYY, hh:mm A zz';
```

Comment:
If this variable is not read by the HTML file, you can move it to outside the `export class` definition. That way you don't need `this.TIME_FORMAT`.